CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ pull request if there was one.
 
 ### Added:
 
+- [Enable Druid In-filter in Fili](https://github.com/yahoo/fili/pull/688)
+    * `DefaultDruidFilterBuilder` is renamed to `DruidOrFilterBuilder`
+    * Implement `DruidInFilterBuilder` which turns list of selector filter generated by `DruidOrFilterBuilder` into a
+      Druid in-filter. The in-filter could substantially shorten druid query, making the query more sharable and
+      readable.
+    * Fili now uses the `DruidInFilterBuilder` as the "Default" Druid query filter builder, in stead of the old
+      `DruidOrFilterBuilder`
```

Comment:
Why change the default? Is it faster? Less memory? Easier to reason about? At least a _little_ of the reason in the changelog would be good.

This is a good description of the change, but is it always the case that an IN filter is more performant than an OR filter? Should it _always_ be the default, or do we need to be a little smarter than that in order to not make queries with only a small number of filters slower?

Reply:
I believe that it's always TERSER.  I'm pretty sure it unpacks to the same logic in Druid historical nodes.  But this packs the query payload tighter, and in proportion to the number of filters being applied.

Reply:
Including this "why" in the commit message or the changelog would be good so that someone else doesn't come along and ask the same question. Or at least when they do, we can point them somewhere to answer the question.